interface/public/messages.options

```diff
@@ -99,10 +99,14 @@ EthereumSignTx.address_n		max_count:8
 EthereumSignTx.nonce			max_size:32
 EthereumSignTx.gas_price		max_size:32
 EthereumSignTx.gas_limit		max_size:32
-EthereumSignTx.to			max_size:20
+EthereumSignTx.to			    max_size:20
 EthereumSignTx.value			max_size:32
 EthereumSignTx.data_initial_chunk	max_size:1024
 EthereumSignTx.to_address_n		max_count:8
+EthereumSignTx.chain_id         max_size:32
```

Comment:
We should get a c formatter we all use before the next release

Reply:
Added retrospective topics on the sprint board